db/schema.rb

```diff
@@ -68,6 +68,7 @@
     t.datetime "updated_at"
     t.integer  "course_id",          limit: 4
     t.string   "display_name",       limit: 255
+    t.integer  "category_id",        limit: 4
```

Comment:
Can you update your database so that this doesn't show up?  Running a rake db:reset with the current develop db/schema.rb might do it, but it's really annoying that this non-existent column keeps showing up.